libs/scripts/scripts.js

```diff
@@ -38,6 +38,36 @@ export async function loadLCP(blocks) {
   }
 }
 
+export async function fetchLib(type, url = '') {
+  const namespace = `${type}Library`;
+  window.milo = window.milo || {};
+  if (!type || window.milo[namespace]) return;
+  const endpoint = (url !== '') ? `${url}/docs/${type}-library.json` : `${window.location.origin}/docs/${type}-library.json`;
+  const resp = await fetch(endpoint);
+  if (resp.ok) {
+    try {
+      const json = await resp.json();
+      window.milo[namespace] = {};
```

Comment:
We need to have a broader conversation about what we are putting in the window scope. We cannot just start putting random things in without thinking about a longer term future. We should also see if we can do some tests where we don't have to pollute the window scope with this stuff. Importing modules with side-effects should probably be able to handle this use case.